docs/integrations/bridges/cbridge.md

```diff
@@ -61,6 +61,16 @@ After the transaction, you can see
 
 and check out your Metamask!
 
+## Understanding the difference betweeen USDT vs. cUSDT
```

Comment:
I would say:
Understanding the difference between USDT vs bridgedUSDT
We also may have other bridges coming to EVM that will have bridged USDT.